src/main/scala/redis/commands/Clusters.scala

```diff
@@ -13,4 +13,7 @@ trait Clusters extends Request {
 
   def clusterSlots(): Future[Seq[ClusterSlot]] = send(ClusterSlots())
 
+  def clusterInfo(): Future[scala.collection.immutable.Map[String, String]] = send(ClusterInfo())
```

Comment:
I think you should be able to use the `Map` alias instead of the full class path `scala.collection.immutable.Map`?